utils/generate_types/overrides.d.ts

```diff
@@ -343,12 +342,8 @@ export type AndroidKey =
   'Copy' |
   'Paste';
 
-export const chromium: BrowserType;
-export const firefox: BrowserType;
-export const webkit: BrowserType;
 export const _electron: Electron;
```

Comment:
I guess we can follow up with _electron and _android to follow _newRequest?

Reply:
Not sure if we want to expose them in the docs, do we?